2022-08-05 17:16:50,444 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Configure stats pid to 94365
2022-08-05 17:16:50,444 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Loading settings from /Users/jiaoyizhu/.config/wandb/settings
2022-08-05 17:16:50,444 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Loading settings from /Users/jiaoyizhu/graphworld1/src/notebooks/wandb/settings
2022-08-05 17:16:50,444 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-08-05 17:16:50,445 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-08-05 17:16:50,445 INFO    MainThread:94365 [wandb_init.py:_log_setup():461] Logging user logs to /Users/jiaoyizhu/graphworld1/src/notebooks/wandb/run-20220805_171650-37oqp1xp/logs/debug.log
2022-08-05 17:16:50,445 INFO    MainThread:94365 [wandb_init.py:_log_setup():462] Logging internal logs to /Users/jiaoyizhu/graphworld1/src/notebooks/wandb/run-20220805_171650-37oqp1xp/logs/debug-internal.log
2022-08-05 17:16:50,445 INFO    MainThread:94365 [wandb_init.py:init():495] calling init triggers
2022-08-05 17:16:50,445 INFO    MainThread:94365 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-08-05 17:16:50,446 INFO    MainThread:94365 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 18if5fjs
2022-08-05 17:16:50,448 INFO    MainThread:94365 [wandb_run.py:_finish():1727] finishing run graph-ood/test-project/18if5fjs
2022-08-05 17:16:50,450 INFO    MainThread:94365 [jupyter.py:save_history():435] not saving jupyter history
2022-08-05 17:16:50,450 INFO    MainThread:94365 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-05 17:16:50,450 INFO    MainThread:94365 [wandb_init.py:_jupyter_teardown():393] cleaning up jupyter logic
2022-08-05 17:16:50,450 INFO    MainThread:94365 [wandb_run.py:_atexit_cleanup():1930] got exitcode: 0
2022-08-05 17:16:50,451 INFO    MainThread:94365 [wandb_run.py:_restore():1913] restore
2022-08-05 17:16:50,451 INFO    MainThread:94365 [wandb_run.py:_restore():1919] restore done
2022-08-05 17:16:50,517 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 701
}

2022-08-05 17:16:50,626 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 701
}

2022-08-05 17:16:51,475 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 701
}

2022-08-05 17:16:52,350 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 3460
}

2022-08-05 17:16:52,460 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 3460
}

2022-08-05 17:16:52,567 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:52,675 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:52,781 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:52,889 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:52,994 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:53,097 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:53,207 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2022-08-05 17:16:53,487 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}
local_info {
}

2022-08-05 17:16:54,603 INFO    MainThread:94365 [wandb_run.py:_footer_history_summary_info():3304] rendering history
2022-08-05 17:16:54,604 INFO    MainThread:94365 [wandb_run.py:_footer_history_summary_info():3336] rendering summary
2022-08-05 17:16:54,607 INFO    MainThread:94365 [wandb_run.py:_footer_sync_info():3260] logging synced files
2022-08-05 17:16:54,614 INFO    MainThread:94365 [wandb_init.py:init():548] starting backend
2022-08-05 17:16:54,614 INFO    MainThread:94365 [wandb_init.py:init():552] setting up manager
2022-08-05 17:16:54,620 INFO    MainThread:94365 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-08-05 17:16:54,624 INFO    MainThread:94365 [wandb_init.py:init():558] backend started and connected
2022-08-05 17:16:54,653 INFO    MainThread:94365 [wandb_run.py:_label_probe_notebook():1105] probe notebook
2022-08-05 17:16:54,653 INFO    MainThread:94365 [wandb_run.py:_label_probe_notebook():1115] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-08-05 17:16:54,653 INFO    MainThread:94365 [wandb_init.py:init():636] updated telemetry
2022-08-05 17:16:54,686 INFO    MainThread:94365 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2022-08-05 17:16:55,806 INFO    MainThread:94365 [wandb_run.py:_on_init():1981] communicating current version
2022-08-05 17:16:55,891 INFO    MainThread:94365 [wandb_run.py:_on_init():1985] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-05 17:16:55,891 INFO    MainThread:94365 [wandb_init.py:init():700] starting run threads in backend
2022-08-05 17:16:55,939 INFO    MainThread:94365 [wandb_run.py:_console_start():1961] atexit reg
2022-08-05 17:16:55,940 INFO    MainThread:94365 [wandb_run.py:_redirect():1819] redirect: SettingsConsole.WRAP_RAW
2022-08-05 17:16:55,940 INFO    MainThread:94365 [wandb_run.py:_redirect():1884] Wrapping output streams.
2022-08-05 17:16:55,940 INFO    MainThread:94365 [wandb_run.py:_redirect():1906] Redirects installed.
2022-08-05 17:16:55,941 INFO    MainThread:94365 [wandb_init.py:init():732] run started, returning control to user process
2022-08-05 17:19:04,283 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Configure stats pid to 94365
2022-08-05 17:19:04,284 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Loading settings from /Users/jiaoyizhu/.config/wandb/settings
2022-08-05 17:19:04,284 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Loading settings from /Users/jiaoyizhu/graphworld1/src/notebooks/wandb/settings
2022-08-05 17:19:04,284 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-08-05 17:19:04,284 INFO    MainThread:94365 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-08-05 17:19:04,285 INFO    MainThread:94365 [wandb_init.py:_log_setup():461] Logging user logs to /Users/jiaoyizhu/graphworld1/src/notebooks/wandb/run-20220805_171904-2z9okljf/logs/debug.log
2022-08-05 17:19:04,285 INFO    MainThread:94365 [wandb_init.py:_log_setup():462] Logging internal logs to /Users/jiaoyizhu/graphworld1/src/notebooks/wandb/run-20220805_171904-2z9okljf/logs/debug-internal.log
2022-08-05 17:19:04,285 INFO    MainThread:94365 [wandb_init.py:_jupyter_setup():411] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x12b67da90>
2022-08-05 17:19:04,285 INFO    MainThread:94365 [wandb_init.py:init():495] calling init triggers
2022-08-05 17:19:04,285 INFO    MainThread:94365 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-08-05 17:19:04,285 INFO    MainThread:94365 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 37oqp1xp
2022-08-05 17:19:04,288 INFO    MainThread:94365 [wandb_run.py:_finish():1727] finishing run graph-ood/test-project/37oqp1xp
2022-08-05 17:19:04,288 INFO    MainThread:94365 [wandb_run.py:_atexit_cleanup():1930] got exitcode: 0
2022-08-05 17:19:04,288 INFO    MainThread:94365 [wandb_run.py:_restore():1913] restore
2022-08-05 17:19:04,289 INFO    MainThread:94365 [wandb_run.py:_restore():1919] restore done
2022-08-05 17:19:04,295 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 701
}

2022-08-05 17:19:04,432 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 701
}

2022-08-05 17:19:05,166 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 3454
}

2022-08-05 17:19:05,270 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 701
  total_bytes: 3454
}

2022-08-05 17:19:05,378 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3454
  total_bytes: 3454
}

2022-08-05 17:19:05,489 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3454
  total_bytes: 3454
}

2022-08-05 17:19:05,596 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3454
  total_bytes: 3454
}

2022-08-05 17:19:05,699 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3454
  total_bytes: 3454
}

2022-08-05 17:19:05,991 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3454
  total_bytes: 3454
}

2022-08-05 17:19:06,247 INFO    MainThread:94365 [wandb_run.py:_on_finish():2220] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3454
  total_bytes: 3454
}
local_info {
}

2022-08-05 17:19:07,359 INFO    MainThread:94365 [wandb_run.py:_footer_history_summary_info():3304] rendering history
2022-08-05 17:19:07,359 INFO    MainThread:94365 [wandb_run.py:_footer_history_summary_info():3336] rendering summary
2022-08-05 17:19:07,368 INFO    MainThread:94365 [wandb_run.py:_footer_sync_info():3260] logging synced files
